gpu-service/jobs.py

```
@@ -51,25 +60,36 @@
 
-# Ensure the 'faces' collection exists (vector size 128, cosine distance)
 try:
-    qdrant.get_collection(collection_name="faces")
-except Exception:
-    logger.info("Creating Qdrant collection 'faces'")
-    qdrant.create_collection(
-        collection_name="faces",
-        vectors_config=models.VectorParams(size=128, distance=models.Distance.COSINE)
-    )
+    ds = deeplake.open('s3://...', creds = DEEPLAKE_CREDS)
+    logger.info(f"Loaded existing DeepLake dataset at {DEEPLAKE_PATH}")
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)